File: BT +/btn/btn/b9.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace btn
{
    class b9
    {
        public static void nhapMang(int[] a,int n)
        {
            for (int i = 0; i < n; i++)
            {
                Console.Write("Nhap gt:");
                a[i] = Convert.ToInt32(Console.ReadLine());
            }
            for (int i = 0; i < n; i++)
            {
                Console.Write(a[i]+",");
            }
            Console.WriteLine(" ");
        }
        public static void sxtc(int[] a,int n)
        {
            int tg;
            for (int i = 0; i < n; i++)
            {
                for (int j = i + 1; j < n; j++)
                {
                    if (a[i] > a[j])
                    {
                        tg = a[i];
                        a[i] = a[j];
                        a[j] = tg;
                    }
                }
            }
            Console.WriteLine("3 vi tri thap den cao:");
            Console.WriteLine(a[0] + "," + a[1] + "," + a[2]);
        }
        public static void sxct(int[] a, int n)
        {
            int tg;
            for (int i = 0; i < n; i++)
            {
                for (int j = i + 1; j < n; j++)
                {
                    if (a[i] < a[j])
                    {
                        tg = a[i];
                        a[i] = a[j];
                        a[j] = tg;
                    }
                }
            }
            Console.WriteLine("3 vi tri cao den thap:");
            Console.WriteLine(a[0] + "," + a[1] + "," + a[2]);
        }
    }
}

File: BT +/b9.cpp
#include<iostream>
using namespace std;
void nhapMang(int n,int a[])
{
	for(int i=0;i<n;i++){
		cin>>a[i];
	}
		for(int i=0;i<n;i++){
		cout<<a[i]<<"\t";
	}
	cout<<endl;
}
void sxtc(int n,int a[]){
	for(int i=0;i<n;i++){
		for(int j=i+1;j<n;j++){
			if(a[i]>a[j]){
				swap(a[i],a[j]);
			}
		}			
	}
	cout<<"3 gia tri dau tu thap den cao:"<<a[0]<<","<<a[1]<<","<<a[2]<<endl;	
}
void sxct(int n,int a[]){
	for(int i=0;i<n;i++){
		for(int j=i+1;j<n;j++){
			if(a[i]<a[j]){
				swap(a[i],a[j]);
			}
		}			
	}
	cout<<"3 gia tri dau tu thap den cao:"<<a[0]<<","<<a[1]<<","<<a[2];	
}
int main()
{
	int n;
	cin>>n;
	int a[100];
	nhapMang(n,a);
	sxtc(n,a);
	sxct(n,a);
}

